ReactWindows/ReactNative.Net46/packages.config

```diff
@@ -1,14 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Facebook.CSSLayout" version="1.1.1" targetFramework="net46" />
```

Comment:
>Facebook.CSSLayout [](start = 15, length = 18)

We may have talked about this before, but we're using version 2.0.1-beta right now for Facebook.CSSLayout.